code/datums/manufacturing.dm

```diff
@@ -244,6 +271,15 @@ ABSTRACT_TYPE(/datum/manufacture)
 	create = 1
 	category = "Miscellaneous"
 
+/datum/manufacture/bottle
```

Comment:
These are already available from glass dispensers. I don't think they need to be made in general fabs as well.

Reply:
I think that should be left for another PR, as bottles are already something in the General Manufacturers. But I could also just remove them from General Manufacturers as well, and a few other weird items that are readily available in another forms but are in manufacturers, like patches.

Reply:
It is best to only include stuff that is actually used in your PR, to avoid clutter. If someone has a good reason to add a glass bottle manufacturer datum in the future, its not like its hard to do.

Reply:
No, I meant that I didn't add the glass bottle datum. It was already there, and GitHub flagged it as new code for some reason.